lib/ui/text.dart

```diff
@@ -770,6 +770,7 @@ Int32List _encodeParagraphStyle(
   StrutStyle strutStyle,
   String ellipsis,
   Locale locale,
+  bool ellipsizedByCharacter,
```

Comment:
I would expose this as an enum, not a bool, so that we can extend it further in the future.